php/getOrderMessages.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * This example script gets information on products and on its related articles from the Tradebyte REST API.
```

Comment:
wrong description for this file

Reply:
Corrected on 0afb26abf53e3882e3499f97dabdb5dd511c6ed2